app/javascript/components/zone-form/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './zone-form.schema';
+import miqRedirectBack from "../../helpers/miq-redirect-back";
+import { http } from '../../http_api';
+
+const ZoneForm = ({recordId}) => {
+    const [{initialValues, isLoading}, setState] = useState({ isLoading: !!recordId });
+
+    const cancelUrl = recordId ? `/ops/zone_edit/${recordId}?button=cancel` : '/ops/zone_edit/?button=cancel';
+
+    const onSubmit = ({href, id, created_on, updated_on, actions, authentications, ...data}) => {
```

Comment:
I don't know why the submit contains these values, they should not be there as they're not part of the form, even if we set them as initialValues. Will get back to you after investigating futher ...